### DISCUSSION TITLE
multi-maven project dependencies are not build

### DISCUSSION THREAD

**Discussion by flashpixx** [quarkusio/quarkus#39384] [Q&A]
Hello,

I have setup a multi-maven project some services based on Quarkus and some library packages without Quarkus dependencies. I'm using native build via container and IÄm using Intellij Professional for development.

My service has got a reference to my library dependencies, but if I build the service i run into the issue that my library dependencies could not be resolved. I try to run `mvn --also-make package` and it seems to work, but how can I run the `quarkus:dev` goal with `also-make`, so that all dependencies of my service will be build?

Thanks a lot

[a comment by an automated account is not shown]

**Reply by aloubyansky** [reply]
Typically, by adding `--also-make compile` to the command line.
